### PR TITLE
Rake task to convert measures; initial work for frontend development

### PR DESCRIPTION
**Pull request by hackrm**
Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1952
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. N/A
- [x] Tests are included and test edge cases 
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable) N/A
- [x] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) ) N/A
- [x] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 
- [x] Automated regression test(s) pass N/A

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

**Review by zlister — COMMENTED** [review]
I think there should probably be a test for these two rake tasks so that coverage doesn't go down.

**Comment by hackrm** [comment]
@zlister - there may be an issue with the previously created test "export single patient from account" in the 'export_fixtures_test.rb' file. After running that test file, please comment on whether or not you notice the following: (1) a large number of fixtures were deleted and created & (2) a directory named 'patient.back' is now located in 'test/fixtures/patients' and 'spec/javascripts/fixtures/json/patients'

File: lib/ext/patient.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module CQM
+  class Patient
+    belongs_to :user
+    scope :by_user, ->(user) { where({'user_id'=>user.id}) }
+    scope :by_user_and_hqmf_set_id, ->(user, hqmf_set_id) { where({'user_id'=>user.id }) }
```
Comment (lukoktonos) [inline lib/ext/patient.rb]: This doesn't use the hqmf_set_id anymore.  I think you can do something like `{measure_ids: hqmf_set_id}`

**Review by zlister — APPROVED** [review]

**Review by lukoktonos — APPROVED** [review]

**Review by zlister — APPROVED** [review]

**Review by zlister — APPROVED** [review]
